libraries/hatch-server-user-management/src/UserInfoRequest.ts

```diff
@@ -0,0 +1,70 @@
+import {BasicRouteParams} from '@launchtray/hatch-server-middleware';
+import {containerSingleton, Logger} from '@launchtray/hatch-util';
+import cookie from 'cookie';
+import {AUTH_ACCESS_TOKEN_COOKIE_NAME} from './constants';
+import AWSCognitoClient from './AWSCognitoClient';
+import {UserInfo} from './UserServiceClient';
+
+@containerSingleton()
+export default class UserInfoRequest {
+  private userInfo: UserInfo | undefined;
+  private logger: Logger;
+  
+  constructor(public readonly params: BasicRouteParams, private readonly userService: AWSCognitoClient) {
+    this.logger = params.logger;
+  }
+  
+  public async getUserInfo() {
+    if (!this.userInfo) {
```

Comment:
Let's get out of the habit of using ! for null checking, even thought it's harmless in this case. Explicit `!= null` (one equals sign) covers undefined and null, but will be safer for false, 0, '', etc.

Reply:
Updated